src/main/java/me/zeroeightsix/kami/util/VectorUtil.java

```diff
@@ -0,0 +1,33 @@
+package me.zeroeightsix.kami.util;
+
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Vec3d;
+
+import java.util.List;
+
+public class VectorUtil {
+
+    public static List<Vec3d> getVecsInArea(Vec3d pos1, Vec3d pos2) {
+        List<Vec3d> returnList = null;
```

Comment:
Also when you do initialise it if you pass it the default size as a parameter it will speed this up a lot as it won’t need to reallocate the array size. The size will be `(maxX - minX) * (maxY-minY) * (maxZ-minZ)`

Reply:
done on local

Reply:
question: will it fuck up if one of the min xyz values are negative